internal/outofband/bmc_helpers.go

```diff
@@ -146,14 +146,26 @@ func (b *bmc) with(provider string) *bmclib.Client {
 
 	if !slices.Contains(b.availableProviders, provider) {
 		b.logger.WithFields(
-			logrus.Fields{"require": provider, "available": strings.Join(b.availableProviders, ",")},
-		).Trace(funcName + ": required provider not in available list")
+			logrus.Fields{
+				"vendor":    b.asset.Vendor,
+				"model":     b.asset.Model,
+				"required":  provider,
+				"available": strings.Join(b.availableProviders, ","),
+				"caller":    funcName,
+			},
+		).Trace("required provider not in available list")
```

Comment:
Is it appropriate for this to just be at `TRACE`? It feels more serious.

Reply:
since this tends to add a lot of noise its a Trace. I'll look at if it can be turned up - in most cases the actual error would bubble up, this is mainly for debugging.